src/style.css

```diff
@@ -0,0 +1,91 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 400;
+}
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  background: #fff;
+  font-weight: 300;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.form {
+  margin-top: 20px;
+  position: relative;
+}
+
+.input {
+  font-family: inherit;
+  font-weight: 400;
+  font-size: 16px;
+  width: 100%;
+  height: 70px;
```

Comment:
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)

Reply:
Hey! Could you please clarify what exactly I should fix here ? I don`t understand  :(

Reply:
I sent you links with code style in CSS
Please, choose one of these code styles & order your styles in your CSS files

Reply:
it's not a fix but rather a suggestion what you can use in the future